docs/integrations/sources/google-analytics-data-api.md

```diff
@@ -264,6 +264,7 @@ The Google Analytics connector is subject to Google Analytics Data API quotas. P
 
 | Version | Date       | Pull Request                                             | Subject                                                                                |
 | :------ | :--------- | :------------------------------------------------------- | :------------------------------------------------------------------------------------- |
+| 2.4.3   | 2024-06-03 | [38884](https://github.com/airbytehq/airbyte/pull/38884)     | Make lookback window configurable.                                                     |
```

Comment:
Can you also update the documentation in the connector configuration showing this is configurable now?

Reply:
sure!

Reply:
https://github.com/airbytehq/airbyte/pull/38884/commits/625a1541ae801c122bfd41ec0bcb667c779f2f7a